src/synth.ts

```diff
@@ -1,10 +1,27 @@
 // Classic variants are from Scale Workshop 2, they're softer on SW3.
 
 import { AperiodicWave } from 'sw-synth'
-import { centsToValue, valueToCents } from 'xen-dev-utils'
+import { centsToValue, sum, valueToCents } from 'xen-dev-utils'
 import { ceilPow2 } from './utils'
 import { computed, type ComputedRef } from 'vue'
 
+import TIMBRES from '@/timbres.json'
+
+type Spectrum = number[]; // Just its partials
```

Comment:
No comment or `// Just the ratios of the frequencies of the partials`. (It's passed to aperiodic-oscillator methods so context is enough docs IMO.)